source/iterators.tex

```diff
@@ -333,6 +333,40 @@
     constexpr insert_iterator<Container>
       inserter(Container& x, ranges::iterator_t<Container> i);
 
+  // \ref{const.iterators}, constant iterators and sentinels
+  template<@\libconcept{indirectly_readable}@ I>
+    using iter_const_reference_t = @\seebelow@;           // freestanding
+  template<class Iterator>
+    concept @\exposconcept{constant-iterator}@ = @\seebelow@; // \expos
+
+  template<class Iterator>
+    class basic_const_iterator;                         // freestanding
+
+  template<class T, @\libconcept{common_with}@<T> U>
+  struct common_type<basic_const_iterator<T>, U> {
+    using type = basic_const_iterator<common_type_t<T, U>>;
+  };
+  template<class T, @\libconcept{common_with}@<T> U>
+  struct common_type<U, basic_const_iterator<T>> {
+    using type = basic_const_iterator<common_type_t<T, U>>;
+  };
+  template<class T, @\libconcept{common_with}@<T> U>
+  struct common_type<basic_const_iterator<T>, basic_const_iterator<U>> {
+    using type = basic_const_iterator<common_type_t<T, U>>;
+  };
```

Comment:
Missing `// freestanding` comments. See https://github.com/cplusplus/draft/pull/5713/files#diff-015716b0f124dba6acb7cdf24285de5d954d0958ca8abbb2ad82a015d1852a9eR470 for example. (And maybe https://github.com/cplusplus/draft/pull/5713#discussion_r939710891).

Reply:
if the primary template is marked "freestanding", any partial specialization should have implicitly that treatment. I'm not convinced it needs a separate comment.

Reply:
> if the primary template is marked "freestanding", any partial specialization should have implicitly that treatment.

There's no wording to suggest that happens. If that were the case, many specializations of `hash` would be missing `// not freestanding`.

Reply:
... if the class for which a template is specialized is freestanding (otherwise freestanding would present an inconsistent view of the world).

But I guess that's too complicated to make implicit.

Reply:
fixed